hphp/runtime/vm/jit/abi.h

```diff
@@ -78,25 +78,27 @@ PhysReg rret_type();
  */
 PhysReg rret(size_t i = 0);
 PhysReg rret_simd(size_t i);
-PhysReg rret_indirect();
 
 /*
  * Native argument registers.
  */
 PhysReg rarg(size_t i);
 PhysReg rarg_simd(size_t i);
+PhysReg rarg_ind_ret(size_t i);
 
 /*
  * Number of available argument registers.
  */
 size_t num_arg_regs();
 size_t num_arg_regs_simd();
+size_t num_arg_regs_ind_ret();
 
 /*
  * RegSet for a call with `n' arguments.
  */
 RegSet arg_regs(size_t n);
 RegSet arg_regs_simd(size_t n);
+RegSet arg_regs_ind_ret(size_t n);
```

Comment:
We can kill `arg_regs_simd()` too, but for this diff, just nix `arg_regs_ind_ret()`.